### ISSUE TITLE
IOS 9.0.2 not listing my Issues

### ISSUE THREAD

**Issue by panshul007** [thedillonb/CodeHub#142]
Hey,

It is amazing App but has some bugs, maybe due to some version changes,
Currently I am not able to see any issues in the App. The list is simply empty.

**Comment by thedillonb** [comment]
Hey @panshul007, where are you going to see your issues? Are you clicking the "Issues" button in the slide-out menu? Or are you going to a repository and clicking "Issues"? The two ways of looking at issues are quite different. The former, clicking "Issues" in the slide-out menu, are issues that are assigned to you or belong to a repository you own. Where clicking "issues" once your in a repository are issues just for that repository.

**Comment by panshul007** [comment]
Hey @thedillonb I access the `Issues` from the side menu. The `closed` tab on the `issues` page shows me my recently closed issues, but the `open` tab is blank.

**Comment by thedillonb** [comment]
Hey @panshul007, Hey @panshul007, if you go to one of your repositories in CodeHub and go to `Issues` do you see issues then?

**Comment by panshul007** [comment]
Yes I can see the issues when I go to organizations-> repositories-> some repository-> issues
I can see all of them.
